File: 1-iniciante/1041/1041.c
#include <stdio.h>

int main() {
    double x, y;
    scanf("%lf%lf", &x, &y);

    if (x == 0 && y == 0) printf("Origem\n");
    else if (x == 0) printf("Eixo Y\n");
    else if (y == 0) printf("Eixo X\n");
    else if (x > 0) {               // x positivo
        if (y > 0) printf("Q1\n");  // x positivo, y positivo
        else printf("Q4\n");        // x positivo, y negativo
    } else if (x < 0) {             // x negativo
        if (y > 0) printf("Q2\n");  // x negativo, y positivo
        else printf("Q3\n");        // x negativo, y negativo
    }
    return 0;
}

File: 1-iniciante/1041/1041.lua
x, y = io.read("*n","*n")

if x == 0 and y == 0 then print ("Origem")
elseif x == 0 then print ("Eixo Y")
elseif y == 0 then print ("Eixo X")
elseif x > 0 then				-- x pos
	if y > 0 then print ("Q1")	-- x, y pos
	else print ("Q4") end		-- x pos y neg
elseif x < 0 then				-- x neg
	if y > 0 then print ("Q2")	-- x neg, y pos
	else print ("Q3") end		-- x neg, y neg
end